src/components/molecules/AppHeaderNav.vue

```diff
@@ -0,0 +1,19 @@
+<script setup lang="ts">
+import { inject } from 'vue'
+
+import { providerFunctionName } from '@/constants/constants'
+
+import AppTitle from '@/components/atoms/AppTitle.vue'
+
+const applicationTitle = inject<string>(providerFunctionName.applicationTitle, 'appName')
```

Comment:
what you try to do using this? you can construct string using `${}` and import/export it or send via props. does you really need use  inject?

Reply:
It was a part of learning process. Tried to use injection to look how it'll work.